README.adoc

```diff
@@ -311,45 +319,45 @@ and disable tracing of the `listContents()` JAX-RS method:
+{inv-url} URL and check your Zipkin server. You see a new trace record that is just one span long for
+the remaining `list()` method in the `InventoryManager` class:
 
 image::/guides/draft-guide-microprofile-opentracing/resources/disable-tracing.png[png][Disable tracing, width=100%]
 
 
 === Injecting a custom Tracer object
 
 The MicroProfile OpenTracing specification also makes the underlying OpenTracing `Tracer` instance
-available for use. The configured `Tracer` can be accessed by injecting it using the `@Inject` annotation
-from the CDI API.
+available for use. You can access the configured `Tracer` by injecting it into a bean using the `@Inject`
+annotation from the CDI API.
 
-Inject the `Tracer` object into `inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java`.
-Then use it to define a new child span in the `get()` method around the `addToInventoryList()` call:
+Inject the `Tracer` object into the `inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java` file.
+Then, use it to define a new child span in the `get()` method around the `addToInventoryList()` call:
 
 [source, Java]
 ----
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=**;!copyright]
 ----
 
-Next, run the `mvn package` command from the `start` directory to rebuild your services. Point to
-{inv-url}/localhost, check your Zipkin server, and you will see a new trace record of four spans long.
-The first two spans for the `getPropertiesForHost()` method in `InventoryResource.java` and the third
-span for the `getProperties()` method in `SystemResource.java`. The last span is the custom span you
-created around the `addToInventoryList()` call.
+Next, run the `mvn package` command from the `start` directory to rebuild your services. Point to the
+{inv-url}/localhost URL and check your Zipkin server. You see a new trace record that is four spans long
+with the first two spans for the `getPropertiesForHost()` method in the `InventoryResource` class and
+the third span for the `getProperties()` method in the `SystemResource` class. The last span is the
+custom span you created around the `addToInventoryList()` call.
 
 image::/guides/draft-guide-microprofile-opentracing/resources/inject-tracer.png[png][Inject tracer, width=100%]
 
-The try block you see here is called a try-with-resources statement, meaning that the `childSpan`
-object will be closed at the end of the statement. It's good practice to define custom spans inside
-such statements since otherwise any exceptions thrown before the span is closed will leak the active span.
+The `try` block that you see here is called a `try-with-resources` statement, meaning that the `childSpan`
+object is closed at the end of the statement. It's good practice to define custom spans inside
+such statements. Otherwise, any exceptions that are thrown before the span is closed will leak the active span.
```

Comment:
Instead of saying something is a "good practice" or "best practice," we prefer to state what advantages it has over other methods of doing something. However, with the more casual tone of the guides, perhaps it's fitting to leave it be.

Reply:
yes lets leave it :)